Feature/refactor codeception robo

vansari opened the pull request:
Hi @DavertMik,

I've refactored and updated the package to the newest version of robo and php etc.
We need this package and I think that nobody should be use a php Version lower then 7.3.

The tests were fixed and extended.

Thanks a lot.

DavertMik commented:
Thanks  for your work @vansari The PR looks good
Could you help with resolving conflicts. After that I'm merging and making a new version

vansari commented:
Hi @DavertMik 
I will do it this evening and write you a Twitter PM if I'm ready with that.

vansari commented:
Hi @DavertMik 

I have fixed the merge conflicts and updated the composer.json and .travis.yml
Tests are back to green ;-) [Forked Tests](https://travis-ci.com/github/vansari/robo-paracept)

Regards
